.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -55,7 +55,7 @@ steps:
       . /usr/share/miniconda/etc/profile.d/conda.sh
       conda activate CB
       cd ..
-      ./s/conda-recipe/run_test.sh
+      ./s/conda-recipe/run_test.sh --json-report

```

Comment:
Since we run the CI tests with the json reports, do we do anything with them? (should be stored in an artifact?)

Reply:
No, they don't go anywhere - it was just added to verify that the command works and the files are produced.

The reports might get quite big though (>20mb) so not sure if it's appropriate to upload them somewhere.

Reply:
If we generate them but don't use them, then its a test of CI and not of the codebase.  I don't think its necessary to run it on every CI job, unless you plan on removing these lines before merging.  If you really want it in, I would run it in the nightly. https://github.com/intel/scikit-learn-intelex/blob/main/.ci/pipeline/nightly.yml#L99 It will show up on main the next day.

Reply:
Thanks for the suggestion. I see the nightly jobs run the commands from this same file - do you suggest to remove the argument `--json-report` from the command in `.github/workflows/ci.yml` then, or is there some other way to make it triggerable for the nightly job but not for the others?

Reply:
I think you may have to set a new azure pipelines variable in ci.yml and nightly.yml that the underlying linux and win yamls will see and trigger from (I hope thats okay).